sdk/eventgrid/azure-messaging-eventgrid/swagger/main.tsp

```diff
@@ -0,0 +1,295 @@
+import "@typespec/http";
```

Comment:
Interesting does this get duplicated over from the specs repo?

Reply:
Will resolve this after beta.